modules/storage-persistent-storage-azure-file-pvc.adoc

```diff
@@ -0,0 +1,41 @@
+// Module included in the following assemblies:
+//
+// * storage/persistent-storage/persistent-storage-azure-file.adoc
+
+[id="create-azure-file-pvc_{context}"]
+= Create the Azure File PersistentVolumeClaim
+
+A PersistentVolumeClaim represents a request for storage by a user or
+application. We will create a PersistentVolumeClaim so that Pods
+will have access to the underlying volumes.
+
+.Prerequisites
+
+* A PersistentVolume exists that references the Azure File share.
+
+.Procedure
+
+* Create a PersistentVolumeClaim that maps to the PersistentVolume
+previously created.
++
+[source,yaml]
+----
+apiVersion: "v1"
+kind: "PersistentVolumeClaim"
+metadata:
+  name: "claim1" <1>
+spec:
+  accessModes:
+    - "ReadWriteOnce"
+  resources:
+    requests:
+      storage: "5Gi" <2>
+  storageClassName: azure-file-sc <3>
+  volumeName: "pv0001" <4>
+----
+<1> The name of the PersistentVolumeClaim.
+<2> The size of this PersistentVolumeClaim.
+<3> The name of the StorageClass used to provision the PersistentVolume.
+This must match the previously defined PersistentVolume's StorageClass.
```

Comment:
s/This must match the previously defined PersistentVolume's StorageClass./Specify the StorageClass that the PersistentVolume that references that Azure File share uses.

Reply:
I don't like using `that` three times in the same sentence, although it is a bit awkward since we're creating a chain of references. I've reworded the sentence in an attempt to convey the same idea.